### PR TITLE
Fixes #256 - Integrate eslint

### LINKED ISSUE #256: Integrate eslint
**Actual Behaviour**

npm test doesnt evoke eslint

**Expected Behaviour**

Use eslint for testing purposes

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by uday96**
Fixes issue #256 

**Changes:**
 - Integrated eslint for testing

**Link to deployment for testing:**  http://udaytheja.surge.sh/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,16 +40,23 @@
...
+    "eslint-plugin-jsx": "0.0.2",
+    "eslint-plugin-react": "^7.0.1",
     "match-media": "^0.2.0",
+    "react-scripts": "1.0.7",
     "react-test-renderer": "^15.6.1"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "lint-js": "eslint *.js \"src/**/*.js\"",
```
Comment (AnshulMalik) [inline package.json]: This worked for me, don't know why it's not working here, but you can try specifying path to local eslint package `./node_modules/...`

**Comment by AnshulMalik** [comment]
Looks like finally eslint is happy

**Comment by uday96** [comment]
@AnshulMalik Yes, Finally. Thanks :smile:

**Comment by uday96** [comment]
@rishiraj824 @isuruAb @madhavrathi @AnshulMalik @dynamitechetan @saurabhjn76 Please review. I have removed the camelCase rule as the data being sent from the server is not in camelCase for few endpoints.

File: .travis.yml [hunk 2]
```diff
@@ -1,27 +1,21 @@
...
-
 script:
-  - npm run test
+  - npm test
 
-after_success:
-  - bash ./deploy.sh
+deploy:
+  provider: script
+  script: "./deploy.sh"
```
Comment (AnshulMalik) [inline .travis.yml]: I like this

**Comment by saurabhjn76** [comment]
Looks good Please resolve conflicts

**Pull request merged**